swagger/medco-connector.yml

```diff
@@ -103,6 +103,20 @@ paths:
       security:
         - medco-jwt: ["medco-explore"]
 
+  /node/status:
+    get:
+      tags:
+        - "medco-node"
+      summary: "Get info about node status."
+      operationId: "getStatus"
+      responses:
+        200:
```

Comment:
Here should be added explicitly other error codes. At least generic 500, maybe other it it makes sense.

Reply:
Given the way it is implemented, this API is supposed to always return a 200 code, I do not see how and why would it return a 500 (or any other) code, so I think that a generic default response is enough to cover all unexpected errors. Tell me if I am missing something.

Reply:
(see previous answer)